doc/active_docs/Service Management API (on-premises).json

```diff
@@ -308,7 +290,7 @@
               "name": "user_id",
               "dataType": "string",
               "paramType": "query",
-              "description": "User id. String identifying an end user. Required only when the application is rate limiting end users. The End User plans feature is not available in all 3scale plans."
+              "description": "User id. String identifying an end user. Required only when the application is rate limiting end users. ."
             },
```

Comment:
Remove the whole user_id attribute :smile: Btw, I see that is it not defined anywhere in the code and only in the jsons. I think that you removed it completely in the other json: https://github.com/3scale/porta/pull/1648/files#diff-c693297ce0ff418d3a1e8d2c9fdfa5f8L216
And that is correct, but u should do the same here :smile: